### ISSUE TITLE
Docs: The link in the `EmojiType` for `ImageOptions` in `og` is outdated

### ISSUE THREAD

**Issue by 1chooo** [vercel/next.js#70949]
### What is the documentation issue?

https://github.com/vercel/next.js/blob/64c414fda59d9c2f4b244de41c86df35d81de3f7/packages/next/src/compiled/%40vercel/og/types.d.ts#L36-L43

`@link https://github.com/vercel/og#emoji` is outdated.

### Is there any context that might help us understand?

Goto https://github.com/vercel/og#emoji and we will find the 404 error message.

<img width="1280" alt="Screenshot 2024-10-08 at 3 53 28 PM" src="https://github.com/user-attachments/assets/ed56665f-2370-4dfb-8f21-b6cafd67e1b6">

### Does the docs page already exist? Please link to it.

There is still no any example in https://nextjs.org/docs/app/api-reference/functions/image-response, but it's the related page.

**Comment by samcx** [comment]
@1chooo Yes, the repo is no longer public. We will update this look very soon!

**Comment by icyJoseph** [comment]
This was fixed, by removing the reference. Closing the issue.
